tests/webidl/post.js

```diff
@@ -1,4 +1,8 @@
 
+process.exit = (e) => {
```

Comment:
probably better to avoid `=>` notation as not all JS engines may support it, and some test environments may run such older VMs.